usl/usl_kevm.h

```diff
@@ -0,0 +1,241 @@
+#ifndef USL_KEVM_H
+#define USL_KEVM_H
+
+#include <cstdint>
+#include <ctime>
+#include <optional>
+#include <string>
+#include <vector>
+
+namespace usl_kevm {
+
+// Common data types
+using bytes_t = std::string;
+using nonce_t = std::uint64_t;
+using ether_t = std::uint64_t;
+using gas_t = std::uint64_t;
+using hash_t = std::uint64_t;
+using account_address_t = std::uint64_t;
+using storage_key_t = std::uint64_t;
```

Comment:
Storage keys and values are evm words, which is a u256

Reply:
added comment